src/flightlog_parser.js

```diff
@@ -1790,22 +1792,18 @@ export function FlightLogParser(logData) {
                     if (frameAccepted) {
                         //Update statistics for this frame type
                         frameTypeStats.bytes += lastFrameSize;
-                        frameTypeStats.sizeCount[lastFrameSize]++;
-                        frameTypeStats.validCount++;
+                        ++frameTypeStats.sizeCount[lastFrameSize];
+                        ++frameTypeStats.validCount;
```

Comment:
Would prefer postfix increments over using prefix increment operators

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Increment

Reply:
> Would prefer postfix increments over using prefix increment operators
A habit... This has not matter in this case, but for one code style - Ok, one moment.